contents/handbook/growth/sales/customer-onboarding.md

```diff
@@ -1,37 +1,116 @@
+
+### Structure
+
+1. Review goals and structure of this session 
+2. Review key concepts:
+   * Acquisition
+   * Activation
+   * User Identification
+   * Cohorts
+   * Groups
+   * Privacy / opt out capture
+3. Have customer demo their app to you, focusing on where the above information is captured
+   * During the demo agree where Acquisition/Activation/Identification take place
+   * Get the CSS selectors and pages of any items to opt-out of capture
+   * Agree any additional properties that need to be captured
+4. Recap and agree the tracking and other implementation details
+5. Agree the timeline to have tracking implemented and set up the following call (ideally 3 days after capture is implemented)
+
+### Deliverables
+
+1. A partially filled-in tracking plan detailing Activation and Acquisitoon
+2. A code snippet showing them how to implement tracking for their product (including Identification and Groups if applicable)
+3. Elements and pages to add opt-out-capture to
+
+## Day 3 - Using PostHog
+
+The aim of this call is to get the customer familiar with navigating PostHog as well as:
+
+* Defining Actions
+* Defining Cohorts
+* Creating Insights
+* Creating Dashboards
+* Finding Recordings
+
+As much as possible the customer should be sharing their screen and driving the session, by teaching them to fish they become comfortable and self-sufficient with PostHog.
+
+### Prerequisites
+
+Tracking should be set up in line with what was shared after the previous call.
+
+### Structure
+
+1. Review goals/agenda
+2. Have them share screen and guide them through:
+
+   1. Live events
+   2. Creating actions
+   3. (Optionally if using Autocapture) the toolbar
+   4. Creating cohorts
+   5. Creating their Acquisiton trend insight
+   6. Creating their Activation funnel
+   7. Adding insights to a dashboard
+   8. Navigate from dashboard to insight to recordings 
+3. Note any inconsistencies or missing tracking information and plan to follow-up to help get that set up 4
+4. Show them the billing page and their projected usage (pricing discussion)
+
+### Deliverables
+
+1. Updated tracking guidance based on issues discovered in the guided demo
+2. Updated pricing quote based on volume
+
+## Next Steps
+
+Every trial should have an end date by which time we expect the customer to make a decision on whether PostHog is right for them.  If they need more time we first need to understand what they've not seen so we can proactively help them see everything they need to do make a decision (within reason).
+
+If they do become a customer (yay!) then we should agree a regular check in call cadence with them from the start (it's much harder to do after they are in the steady state).
```

Comment:
Do we want to add any detail about this step? How do we send the proposal, how do we get them to sign etc?

Reply:
Likely for another handbook page